specs/trip_dispatch_spec.rb

```diff
@@ -111,4 +106,23 @@
       expect(passenger.trips).must_include trip
     end
   end
+
+  describe "request trip method" do
+    it 'will populate a new trip with the necessary elements' do
+
+      dispatcher = RideShare::TripDispatcher.new(USER_TEST_FILE, TRIP_TEST_FILE, DRIVER_TEST_FILE)
```

Comment:
I would love to see some tests around driver availability here. For example, what happens if none of the drivers have status `:AVAILABLE`? What if the only driver who has that status is the passenger on this trip?